### PR TITLE
Ccd 1737 Kustomize gateway-web

### PR DESCRIPTION
**Pull request by nathan-clark**
https://tools.hmcts.net/jira/browse/CCD-1737

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/namespaces/ccd/ccd-api-gateway-web/ccd-api-gateway-web.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  releaseName: ccd-api-gateway-web
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/ccd-api-gateway-web
+  values:
+    nodejs:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/ccd/api-gateway-web:prod-467f957-20210802154948
```
Comment (timja) [inline k8s/namespaces/ccd/ccd-api-gateway-web/ccd-api-gateway-web.yaml:15]:
```suggestion
      image: hmctspublic.azurecr.io/ccd/api-gateway-web:prod-467f957-20210802154948  #{"$imagepolicy": "flux-system:ccd-api-gateway-web"}
```

File: k8s/namespaces/ccd/ccd-api-gateway-web/demo.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+metadata:
+  name: ccd-api-gateway-web
+spec:
+  releaseName: ccd-api-gateway-web
+  values:
+    nodejs:
+      ingressClass: traefik-no-proxy
+      environment:
+        CORS_ORIGIN_WHITELIST: "*"
+        TIMING-ALLOW-ORIGIN: "https://www-ccd.demo.platform.hmcts.net"
```
Comment (timja) [inline k8s/namespaces/ccd/ccd-api-gateway-web/demo.yaml:12]: should should be set either in the base or in your helm chart and only overridden in prod

**Review by timja — APPROVED** [review]
image policy + duplication, also please look at perftest config as its different from other envs (including prod)

**Review by timja — DISMISSED** [review]
meant to request changes

**Comment by timja** [comment]
conflicts

File: k8s/namespaces/ccd/ccd-api-gateway-web/demo.yaml (hunk 2 above)
Comment (nathan-clark) [reply k8s/namespaces/ccd/ccd-api-gateway-web/demo.yaml:12]: @MSancaktutar i assume this should be in helm chart?

Comment (nathan-clark) [reply k8s/namespaces/ccd/ccd-api-gateway-web/demo.yaml:12]: And this in perf: 
```
CORS_ORIGIN_WHITELIST: "*"
TIMING-ALLOW-ORIGIN: "https://www-ccd.perftest.platform.hmcts.net"
UV_THREADPOOL_SIZE: 128
```

Comment (MSancaktutar) [reply k8s/namespaces/ccd/ccd-api-gateway-web/demo.yaml:12]: We'll action clean up pf env vars from env-specific flux files under a separate ticket.

**Comment by MSancaktutar** [comment]
@timja , @cakeben  - could one of you please approve? Many thanks.

**Review by MSancaktutar — DISMISSED** [review]

**Review by timja — DISMISSED** [review]

**Review by MSancaktutar — APPROVED** [review]

**Review by cakeben — APPROVED** [review]

**Pull request merged**